src/main/java/frc/robot/subsystems/shooter_intake/ShooterIntakeIOSim.java

```diff
@@ -0,0 +1,57 @@
+package frc.robot.subsystems.shooter_intake;
+
+import edu.wpi.first.math.MathUtil;
+import edu.wpi.first.math.controller.PIDController;
+import edu.wpi.first.math.system.plant.DCMotor;
+import edu.wpi.first.wpilibj.simulation.FlywheelSim;
+import org.littletonrobotics.junction.Logger;
+
+public class ShooterIntakeIOSim implements ShooterIntakeIO {
+
+    FlywheelSim flywheelSim = new FlywheelSim(DCMotor.getCIM(2), 1.0, 1.0);
+    PIDController controller = new PIDController(0.75, 0.1, 0.0);
```

Comment:
If these are going to be the control gains, we should move them to constants as well